src/reference/sectionC_interpretation.inc

```diff
@@ -281,9 +286,9 @@ Interpretation of ``initial_value`` attributes
 
 #. The :code:`initial_value` attribute of a :code:`variable` element MUST either be a :ref:`real number string<specA_real_number>`, or a :ref:`variable reference<specC_variable_reference>`.
 
-#. The conditions when initial values hold are (by design) not defined in a :ref:`CellML model<specA_cellml_model>` document.
+#. The conditions when initial values hold are (by design) not defined in a :ref:`CellML model<specA_cellml_model>`.
 
-#. Where the :code:`initial_value` attribute has a :ref:`real number value<specA_real_number>`, it SHALL be interpreted as a statement that the variable on which the attribute appears is equal to that real number value, under the conditions when the initial value holds.
+#. Where the :code:`initial_value` attribute is a :ref:`real number string<specA_real_number>`, it SHALL be interpreted as a statement that the variable on which the attribute appears is equal to that real number value, under the conditions when the initial value holds.
```

Comment:
We previously replaced "Where" with "If" at the beginning of a clause. Why don't we do it here (and below, and maybe elsewhere too?)?

Similarly, there are been corrections that read "..., [then] it SHALL..." So, why no "then" here?

IOW, some recent changes have the risk of making our spec inconsistent. So, need to check that.

Reply:
I think it's going to be easiest to merge this and other PRs in first then check for consistency?

Reply:
Anyway. Here a "where" is good. We've just had a sentence talking about _all_ intial values, so now we say "where" to indicate a specific one

Reply:
And we don't need a spec that has super formulaic sentences. As long as it's unambiguous.

Reply:
Ok, 2-1, so fine with that. :)